src/parser/php.rs

```diff
@@ -330,6 +355,7 @@ impl PhpParser {
             Ok(plugin_type) => Some(ClassAttribute::Plugin(DrupalPlugin {
                 plugin_id,
                 plugin_type,
+                usage_example: None,
```

Comment:
```suggestion
                usage_example: self.extract_usage_example_from_comment(&node.parent().prev_named_sibling()),
```
Maybe also check that node.parent().prev_named_sibling().kind() is equal to "comment".

Reply:
I check the node.kind() in the extract function now, which takes the command_node argument.
If it's not a "comment" kind, I return None.